### PR TITLE
[hammerhead] Added redscreen kernel patch and xorg config

### PR DESCRIPTION
**Pull request by MartijnBraam**
This makes xorg and xfce4 work on the hammerhead thanks to the changes found by @zhenyolka

### CODE REVIEW

File: aports/device/device-lg-hammerhead/APKBUILD [hunk 1]
```diff
@@ -30,6 +31,8 @@ package() {
 		"$pkgdir"/etc/profile.d/00-udev-v4l-hack.sh
 	install -Dm644 "$srcdir"/bcmdhd.cal \
 		"$pkgdir"/etc/wifi/bcmdhd.cal
+	install -Dm644 "$srcdir"/xorg.conf \
```
Comment (drebrez) [inline aports/device/device-lg-hammerhead/APKBUILD]: @MartijnBraam what about installing the xorg configuration with an `install_if` condition? (similar to [this](https://github.com/postmarketOS/pmbootstrap/blob/7ab85bc4a5ae748b84759d4e12c367e4979db96d/aports/main/postmarketos-base/APKBUILD#L56))

Comment (MartijnBraam) [reply aports/device/device-lg-hammerhead/APKBUILD]: Ohh yes, good idea

File: aports/device/device-lg-hammerhead/APKBUILD [hunk 2]
```diff
@@ -32,8 +33,20 @@ package() {
 		"$pkgdir"/etc/wifi/bcmdhd.cal
 }
 
+x11() {
+	install_if="$pkgname xorg-server"
+	depends="
+		libinput
+		xf86-input-libinput
+		xf86-video-fbdev"
+	mkdir "$subpkgdir"
```
Comment (ollieparanoid) [inline aports/device/device-lg-hammerhead/APKBUILD]: `abuild` requires the `$subpkgdir` to exist, that's why `postmarketos-base` has this in the subpkg functions. But here the subfolder gets created already with `install -D`, so the line is redundant.

File: aports/device/device-lg-hammerhead/APKBUILD [hunk 3]
```diff
@@ -32,8 +33,20 @@ package() {
...
+	install -Dm644 "$srcdir"/xorg.conf \
+		"$subpkgdir"/etc/X11/xorg.conf.d/10-hammerhead.conf
+}
+
 weston() {
 	install_if="$pkgname weston"
+	mkdir "$subpkgdir"
```
Comment (ollieparanoid) [inline aports/device/device-lg-hammerhead/APKBUILD]: `abuild` requires the `$subpkgdir` to exist, that's why `postmarketos-base` has this in the subpkg functions. But here the subfolder gets created already with `install -D`, so the line is redundant.

**Comment by MartijnBraam** [comment]
I've removed the weston pixman type workaround now as it was indeed not needed anymore. I've also added the msm-fb-refresher which I was shure about I've added it already.

**Pull request merged**